src/debugging/DockerDebugConfigurationProvider.ts

```diff
@@ -42,21 +44,14 @@ export class DockerDebugConfigurationProvider implements DebugConfigurationProvi
     }
 
     public provideDebugConfigurations(folder: WorkspaceFolder | undefined, token?: CancellationToken): ProviderResult<DebugConfiguration[]> {
-        const add: MessageItem = { title: l10n.t('Add Docker Files') };
-
-        // Prompt them to add Docker files since they probably haven't
-        /* eslint-disable-next-line @typescript-eslint/no-floating-promises */
-        window.showErrorMessage(
-            l10n.t('To debug in a Docker container on supported platforms, use the command "Docker: Add Docker Files to Workspace", or click "Add Docker Files".'),
-            ...[add])
-            .then((result) => {
-                if (result === add) {
-                    /* eslint-disable-next-line @typescript-eslint/no-floating-promises */
-                    commands.executeCommand('vscode-docker.configure');
-                }
-            });
 
-        return [];
+        return callWithTelemetryAndErrorHandling(
+            'docker-provideDebugConfigurations', //TODO: change this later

```

Comment:
what should we name this? is `docker-provideDebugConfigurations` sufficient?

Reply:
The telemetry name will be prepended with `vscode-docker.` automatically so I think we can at least remove the `docker-`.